packages/react-components/react-table/src/components/TableRow/useTableRow.ts

```diff
@@ -4,6 +4,7 @@ import { useFocusVisible, useFocusWithin } from '@fluentui/react-tabster';
 import type { TableRowProps, TableRowState } from './TableRow.types';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 76 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27884&commitId=ed93fe89b929a239ecc48351b229bf04e1f336ed&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1860 screenshots unchanged, 0 screenshots with different dimensions and 76 screenshots with visible difference.


  <details><summary><b>unknown</b> 76 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Table flex - subtle selection.rest.chromium.png|14059| Changed|
|Table flex - subtle selection.rest.hover unselected row.chromium.png|13840| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|31339| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|54281| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|15303| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|15293| Changed|
|Table layout flex.multiselect (checked) - neutral - Dark Mode.chromium.png|29966| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|45903| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|16198| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|16186| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|33405| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|90837| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|17686| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|17680| Changed|
|Table layout flex.multiselect (mixed) - neutral - Dark Mode.chromium.png|31085| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|75232| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|19586| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|19580| Changed|
|Table layout flex.multiselect - Dark Mode.chromium.png|32708| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|52314| Changed|
|Table layout flex.multiselect - RTL.chromium.png|14263| Changed|
|Table layout flex.multiselect.chromium.png|14255| Changed|
|Table layout flex.single select (checked) - Dark Mode.chromium.png|32626| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|52142| Changed|
|Table layout flex.single select (checked) - RTL.chromium.png|14205| Changed|
|Table layout flex.single select (checked) - brand - Dark Mode.chromium.png|33198| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|90066| Changed|
|Table layout flex.single select (checked) - brand - RTL.chromium.png|15682| Changed|
|Table layout flex.single select (checked) - brand.chromium.png|15675| Changed|
|Table layout flex.single select (checked) - neutral - Dark Mode.chromium.png|31797| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|75732| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|16628| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|16621| Changed|
|Table layout flex.single select (checked).chromium.png|14198| Changed|
|Table layout flex.single select - Dark Mode.chromium.png|32580| Changed|
|Table layout flex.single select - High Contrast.chromium.png|52068| Changed|
|Table layout flex.single select - RTL.chromium.png|14111| Changed|
|Table layout flex.single select.chromium.png|14104| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|31339| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|54281| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|15303| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|15293| Changed|
|Table layout table.multiselect (checked) - neutral - Dark Mode.chromium.png|29966| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|45903| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|16198| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|16186| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|33405| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|90837| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|17686| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|17680| Changed|
|Table layout table.multiselect (mixed) - neutral - Dark Mode.chromium.png|31085| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|75232| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|19586| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|19580| Changed|
|Table layout table.multiselect - Dark Mode.chromium.png|32708| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|52314| Changed|
|Table layout table.multiselect - RTL.chromium.png|14263| Changed|
|Table layout table.multiselect.chromium.png|14255| Changed|
|Table layout table.single select (checked) - Dark Mode.chromium.png|32626| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|52142| Changed|
|Table layout table.single select (checked) - RTL.chromium.png|14205| Changed|
|Table layout table.single select (checked) - brand - Dark Mode.chromium.png|33198| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|90066| Changed|
|Table layout table.single select (checked) - brand - RTL.chromium.png|15682| Changed|
|Table layout table.single select (checked) - brand.chromium.png|15675| Changed|
|Table layout table.single select (checked) - neutral - Dark Mode.chromium.png|31797| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|75732| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|16628| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|16621| Changed|
|Table layout table.single select (checked).chromium.png|14198| Changed|
|Table layout table.single select - Dark Mode.chromium.png|32580| Changed|
|Table layout table.single select - High Contrast.chromium.png|52068| Changed|
|Table layout table.single select - RTL.chromium.png|14111| Changed|
|Table layout table.single select.chromium.png|14104| Changed|
|Table table - subtle selection.rest.chromium.png|14059| Changed|
|Table table - subtle selection.rest.hover unselected row.chromium.png|13840| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>